projects/1.20/assets/embeddium/embeddium/lang/zh_cn.json

```
@@ -0,0 +1,8 @@
+{
+	"embeddium.conflicting_mod": "Embeddium 检测到已安装的模组中有些可能会导致渲染问题或崩溃。建议卸载它们。请参考以下列表。",
+	"embeddium.conflicting_mod_list": "[{2}]",
+	"embeddium.search_bar_empty": "搜索选项……",
+	"embeddium.options.added_by_mod_string": "由下列模组添加: %s",
```

```suggestion
	"embeddium.options.added_by_mod_string": "由下列模组添加：%s",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。